hikari/interactions/command_interactions.py

```diff
@@ -130,11 +136,17 @@ class CommandInteractionOption:
     subcommand or group.
     """
 
+    is_focused: bool = attr.field(default=False, repr=False)
+    """Whether this option is the currently focused option for autocomplete."""
```

Comment:
This doesn't need to be added to the options of normal commands interactions since it just means nothing there, they should just be using different option types